recipe/meta.yaml

```diff
@@ -8,17 +9,24 @@ package:
 
 source:
   url: https://github.com/HowardHinnant/date/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 7a390f200f0ccd207e8cff6757e04817c1a0aec3e327b006b7eb451c57ee3538
+  sha256: {{ sha256 }}
 
 build:
   number: 0
+  run_exports:
+    strong:
+      - libcurl
 
 requirements:
-  build:
+  host:
     - {{ compiler('c') }}
```

Comment:
What if the sections will be:
```
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - cmake
    - ninja
    - libcurl
host:
    - libcurl 
```

Reply:
Is that right? I always thought that the compilers and build tools need to be in the `host` section.

Reply:
No, compilers need to be in `req/build`

Reply:
@skupr-anaconda ok, that's much better! Thank you for the help!